astropy/io/fits/convenience.py

```diff
@@ -408,9 +410,11 @@ def writeto(filename, data, header=None, output_verify='exception',
         ``+warn``, or ``+exception" (e.g. ``"fix+warn"``).  See :ref:`verify`
         for more info.
 
-    clobber : bool, optional
-        If `True`, and if filename already exists, it will overwrite
-        the file.  Default is `False`.
+    overwrite : bool, optional
+        If ``True``, overwrite the output file if exists.
```

Comment:
Should the default value for a parameter (here ``False``) be mentioned in the parameter list, and if so, where (if there is a preference)? Directly following the type and ", optional" line, at the start of the description, or at the end?

The default value is of course glanced immediately from the function definition, but so is the fact that the keyword is optional, and the latter is still explicitly listed in the parameter list.